pkg/utils/experimentHelpers.go

```diff
@@ -8,6 +8,14 @@ import (
 	"k8s.io/klog"
 )
 
+func CreateExperimentList(engineDetails *EngineDetails) *[]ExperimentDetails {
+	var ExperimentDetailsList []ExperimentDetails
+	for i, _ := range engineDetails.Experiments {
+		ExperimentDetailsList = append(ExperimentDetailsList, NewExperimentDetails(engineDetails, i))
+	}
+	return &ExperimentDetailsList
```

Comment:
why are we returning it like this, we can simply return the value instead of reference